config/locales/en/en.yml

```diff
@@ -13,6 +13,15 @@ en:
       person: "Member"
       project_step: "Project Step"
     attributes:
+      basic_project:
+        agents: "Agents"
+        length_months: "Length (months)"
+        name: "Name"
+        primary_agent: "Primary Agent"
+        secondary_agent: "Secondary Agent"
+        signing_date: "Start Date"
```

Comment:
So in the database, it is signing date. But this is a basic project and not a loan. So we are using signing date as start date. Using the language "Signing Date" is not to the spec and does not seem like it would match the intent of a basic project.